### PR TITLE
Indentation redone, ShellCheck done

### PR DESCRIPTION
**Pull request by wdhif**
- I've redone the indentation correctly
- I've run shellcheck on the script and fix the issues
- The exit code when displaying the help should be 0

### CODE REVIEW

File: extract.sh [hunk 1]
```diff
@@ -2,37 +2,45 @@
 # function Extract for common file formats
 
 function extract {
- if [ -z "$1" ]; then
+  if [ -z "$1"  ]; then
```
Comment (per1234) [inline extract.sh]: Was the added space before the closing bracket here and at line 13 intentional?

Comment (wdhif) [inline extract.sh]: Oh no it's actually a mistake, I'll fix this quickly, thanks !

Comment (wdhif) [inline extract.sh]: Also on line 36, might be my editor :/

**Comment by gsayer** [comment]
up

**Comment by xvoland** [comment]
I don't see point in these manipulations. Return code of func help I fixed

**Pull request closed without merging**